packages/web/src/components/edit/fields/visibility/ReleaseDateField.tsx

```diff
@@ -31,29 +31,14 @@ export const ReleaseDateField = () => {
   return (
     <Flex direction='column' gap='l'>
       <Flex gap='l'>
-        <Flex direction='column' gap='s'>
-          <Flex
-            backgroundColor='surface1'
-            border='default'
-            borderRadius='s'
-            ph='l'
-            pv='m'
-            css={(theme) => ({
-              '&:hover': {
-                borderColor: theme.color.border.strong
-              }
-            })}
-          >
-            <DatePickerField
-              name='releaseDate'
-              label={messages.dateLabel}
-              futureDatesOnly
-            />
-          </Flex>
-          {error && (touched || submitCount > 0) ? (
-            <Text color='danger'>{error}</Text>
-          ) : null}
-        </Flex>
+        <DatePickerField
+          name='releaseDate'
+          label={messages.dateLabel}
+          futureDatesOnly
+        />
+        {error && (touched || submitCount > 0) ? (
```

Comment:
is there a world where we want this error text built into datepickerfield?

Reply:
Possibly? This is my first time messing with it. Lemme poke around if its simple to move it in

Reply:
Nice yeah it was fairly simple to move in, like it better for sure